src/probabilities_estimators/dispersion/dispersion.jl

```diff
@@ -0,0 +1,172 @@
+using DelayEmbeddings
+using StatsBase
+
+export Dispersion
+
+"""
+    Dispersion(; s = GaussianSymbolization(c = 5), m = 2, τ = 1, check_unique = true,
+        normalize = true)
+
+A probability estimator based on dispersion patterns, originally used by
+Rostaghi & Azami, 2016[^Rostaghi2016] to compute the "dispersion entropy", which
+characterizes the complexity and irregularity of a time series.
+
+Relative frequencies of dispersion patterns are computed using the symbolization scheme
+`s` with embedding dimension `m` and embedding delay `τ`. Recommended parameter
+values[^Li2018] are `m ∈ [2, 3]`, `τ = 1` for the embedding, and `c ∈ [3, 4, …, 8]`
+categories for the Gaussian symbol mapping. See below for an in-depth description of
+dispersion pattern construction and usage.
```

Comment:
```suggestion
categories for the Gaussian symbol mapping.
```
The description follows immediatelly, so this sentence isn't necessary :)